nativelink-util/src/origin_event_publisher.rs

```diff
@@ -85,7 +84,7 @@ impl OriginEventPublisher {
     }
 
     async fn handle_batch(&self, batch: &mut Vec<OriginEvent>) {
-        let uuid = Uuid::now_v6(&get_node_id(None));
+        let uuid = Uuid::new_v4();
```

Comment:
Looks ok to me, but don't know if anything else was relying on the v6 properties?

Reply:
I actually wanted to go to 7, but let me check for 6 first. This is an error.